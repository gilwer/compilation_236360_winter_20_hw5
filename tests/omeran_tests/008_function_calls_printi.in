// calling functions like printi
enum days { SUN, MON, TUES };

void main() {
  int x = 3;
  printi(x); // 3
  int y;
  printi(y); // 0
  printi(44+22); // 66
  printi((int)MON); // 1
  printi((int)MON + (int)TUES); // 3
}