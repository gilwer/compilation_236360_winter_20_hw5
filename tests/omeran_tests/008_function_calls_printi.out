3
0
66
1
3
